### PR TITLE
Updated Marketing Toolkit to Include the PDF

### PR DESCRIPTION
**Pull request by LaurelC3**
Included the PDF Marketing Toolkit link as we want to also make this available as an option.

Also removed Arian from the Channel Sales contacts.

### CODE REVIEW

File: src/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.markdown [hunk 1]
```diff
@@ -7,6 +7,8 @@ layout: reference
 
 SAP Concur is pleased to welcome you as a partner. We encourage all Concur App Center partners to create comprehensive marketing plans that incorporate PR, social media, digital marketing, events and sales training to help make the partnership a success. This toolkit provides guidelines and templates for referencing the SAP Concur brand and collaborating on marketing efforts.
 
+**The PDF version of this toolkit is [available here](https://developer.concur.com/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.pdf).**
```
Comment (charlieconcur) [inline src/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.markdown:10]: I'm getting ready to delete this PDF as redundant so we probably should not link to it.

File: src/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.markdown [hunk 2]
```diff
@@ -283,7 +285,7 @@ Email content to: [concur_AppCenterMarketing@sap.com](mailto:concur_AppCenterMar
 
 * App Center Alliances: [concur_AppCenterAlliance@sap.com](mailto:concur_AppCenterAlliance@sap.com)
 * App Center Marketing: [concur_AppCenterMarketing@sap.com](mailto:concur_AppCenterMarketing@sap.com)
-* App Center Channel Sales: [aaron.guerra@sap.com (SMN)](mailto:aaron.guerra@sap.com) and [arian.white@sap.com (ENT)](mailto:arian.white@sap.com)
+* App Center Channel Sales: [aaron.guerra@sap.com (SMN)](mailto:aaron.guerra@sap.com) and TBH - ENT
```
Comment (charlieconcur) [inline src/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.markdown:288]: We need to remove individual email names and replace them with distribution lists and no TBH -- just leave out.

**Review by charlieconcur — CHANGES_REQUESTED** [review]

**Comment by LaurelC3** [comment]
Hi Charlie,

Thanks for your help with the updates.

Please don’t delete the PDF – we like the .html version as well, but want to also make the PDF available to the partners because it’s a little more visual than the page.

Let’s leave out the TBH, but Aaron doesn’t have an alias, so need to keep his email in for now. If he does get an alias, I’m ok to make the pull request to do the change.

Best,
Laurel

**Comment by charlieconcur** [comment]
@LaurelC3 Would you like to make modifications to this pull request to remove the link to the (now non-existent) PDF and fix the TBH email address or would you simply like for me to close this pull request? I can rather easily make the email address updates on your behalf if you like.

**Comment by LaurelC3** [comment]
Hi Charlie,

Please close the pull request for now.

I think we should still have the PDF version available as an option, at least for now. We had a partner reach out this morning that the link we’d sent to the Marketing Toolkit PDF was broken and providing a 404 error, so at the very least, we should be re-directing from the old link.

The issue with the HTML page is that partners have to scroll all the way down through a lot of similar-looking text to read the full thing – it’s not really conducive to skimming to easily find the guidance they’re looking for. I have pretty limited bandwidth, so re-doing the whole thing into multiple HTML pages with a similar layout as the PDF and coding them in markdown isn’t something I’d be able to quickly knock out.

Best,
Laurel

Laurel Clippinger | Partner Marketing Manager | SAP Concur App Center

**Pull request closed without merging**